pkg/ocihook/ocihook.go

```diff
@@ -158,6 +158,8 @@ func newHandlerOpts(state *specs.State, dataStore, cniPath, cniNetconfPath strin
 		if err != nil {
 			return nil, err
 		}
+	case nettype.Container:
```

Comment:
Why should we ignore this type?

Reply:
I moved this line to LINE 138 following `case nettype.Host, nettype.None`, It just means no special OCIHook need when `createRuntime` and `postStop`

https://github.com/containerd/nerdctl/blob/master/cmd/nerdctl/run.go#L891-L913

Reply:
SGTM